### ISSUE TITLE
"Property not found"-problem when using seqnum during bulk-import

### ISSUE THREAD

**Issue by sacha-alex** [dhlab-basel/Knora#971]
@benjamingeer

Dear Ben,

  I run into a problem when I am using a seqnum during bulk import. The XML-file seems correct and evaluates against the xsd I got from knora-API's "http://localhost:3333/v1/resources/xmlimportschemas/..." call. Even Knora seems to evaluate the XML-file and starts with the import when it suddenly breaks with the message:

   status":9,"error":"org.knora.webapi.NotFoundException: Property not found: http://www.knora.org/ontology/004D/kuno-raeber#knoraXmlImport__seqnum"

As mentioned before, the XML-file seems correct. Any suggestions?
Please find the necessary files (xml, xsd and ttl) attached.
Many thanks in advance.

Best,
Sascha
[seqnum.zip](https://github.com/dhlab-basel/Knora/files/2302869/seqnum.zip)

**Comment by benjamingeer** [comment]
I will also need the Knora admin data for all these projects, containing the project definitions. Otherwise, GraphDB rejects the ontologies (because they refer to nonexistent projects).

**Comment by benjamingeer** [comment]
Also, in your XML file, can you try using `knoraXmlImport:seqnum` instead of `kuno-raeber:knoraXmlImport__seqnum`?

**Comment by sacha-alex** [comment]
Sorry, I forgot ...

[admin-data.ttl.zip](https://github.com/dhlab-basel/Knora/files/2303045/admin-data.ttl.zip)
[permissions-data.ttl.zip](https://github.com/dhlab-basel/Knora/files/2303046/permissions-data.ttl.zip)

**Comment by sacha-alex** [comment]
Tried the other one with ` knoraXmlImport:seqnum`, but got the following error:

status":11,"error":"org.knora.webapi.BadRequestException: XML import did not pass XML schema validation: org.xml.sax.SAXParseException; lineNumber: 497; columnNumber: 50; cvc-complex-type.2.4.a: Invalid content was found starting with element 'knoraXmlImport:seqnum'


[raeber-data-xmlImport_seqnum.xml.zip](https://github.com/dhlab-basel/Knora/files/2303074/raeber-data-xmlImport_seqnum.xml.zip)

**Comment by benjamingeer** [comment]
Blocked by #972.

**Comment by benjamingeer** [comment]
@sacha-alex I got this error:

```
[ERROR] Dispatcher - Sipi reported an internal server error 500 Internal Server Error - {
   "message": "Submitted file path could not be read: /Users/raeber/data/images/space.png"
}
```

Could you give me a test case that doesn't require Sipi?

**Comment by sacha-alex** [comment]
Sure, here you are.
[raeber-data-small.xml.zip](https://github.com/dhlab-basel/Knora/files/2306973/raeber-data-small.xml.zip)

**Comment by benjamingeer** [comment]
I think this is fixed in #973, could you try the `fix/971-import-seqnum` branch?

There seem to be some other inconsistencies in your test data (both the large and the small version): some links between resources seem to be invalid.

If possible, could you please make a minimal working example (in as few files as possible) that I could use in an automated test?
